frontend/app/ui_components/index.js

```diff
@@ -102,6 +102,7 @@ angular.module('openproject.uiComponents')
     './toggled-multiselect-directive')])
   .directive('toolbar', require('./toolbar-directive'))
   .constant('ESC_KEY', 27)
+  .directive('userField', [require('./user-field-directive')])
```

Comment:
@NobodysNightmare nvm.. will merge as is. we can shift the rename to a separate PR.